src/targets/enemy.js

```diff
@@ -253,22 +252,11 @@ export class Enemy extends Target {
 
     getDamageLevel() {
         if (Object.entries(this.equipment.obj.equipment.WEAPON).length) {
-            const skill = this.equipment.obj.equipment.WEAPON.skill;
-            // Todo: organize so switch isn't needed
-            switch (this.equipment.obj.equipment.WEAPON.skill) {
-                case EQUIPMENT.WEAPON_TYPES.MAGIC:
-                    return calcLevel(characterData.getSkillXp("magic"));
-                    break;
-                case EQUIPMENT.WEAPON_TYPES.RANGED:
-                    return calcLevel(characterData.getSkillXp("ranged"));
-                    break;
-                case EQUIPMENT.WEAPON_TYPES.MELEE:
-                    return calcLevel(characterData.getSkillXp("attack"));
-                    break;
-            }
-        } else {
```

Comment:
This else is needed otherwise unarmed does 0 damage and gives 0 xp

Reply:
[Fixed](https://github.com/EthanVieira/rs-clicker/pull/51/commits/3cc7bbd09b9aa339abf9b002707b19174241c7df)!

Reply:
![image](https://user-images.githubusercontent.com/9402065/113242402-8334bd80-9276-11eb-9d1d-6b825c4ee2b3.png)
Lol

This can be fixed by keeping that `if (object exists) {} else { "unarmed"}` check. That way for the normal one you can still have error checking on it. On that note I think the `Object.entries(obj).length` might be a good utility function to add as well. Can be called `exists(obj)` or objExists(obj)` or something

Reply:
The screenshot is when you aa unarmed

Reply:
Decided to remove log. https://github.com/EthanVieira/rs-clicker/pull/51/commits/b73bf2b6aa7cc726e0d5e6d0c3054dfbdc5bddf7